src/pages/settings/Payments/PaymentMethodList.js

```diff
@@ -128,6 +138,14 @@ class PaymentMethodList extends Component {
         return combinedPaymentMethods;
     }
 
+    /**
+     * @param {*} paymentMethod
```

Comment:
NAB: I think we add types here to the object argument to keep consistency with the prop types

```suggestion
     * @param {Object} paymentMethod
     * @param {String|Number} paymentMethod.methodID
     * @param {String} paymentMethod.accountType
```

Reply:
Sure.